build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -892,6 +884,11 @@ message OutputFile {
 
   // True if file is executable, false otherwise.
   bool is_executable = 4;
+
+  // The contents of the file if inlining was requested. The server SHOULD NOT inline
```

Comment:
Should we be explicit somewhere that the server MAY omit inlining, even if requested, and MUST do so if inlining would cause the response to exceed message size limits? Possibly here (and on other output fields), possibly on the relevant fields in GetActionResultRequest instead.

Reply:
done.